applications/luci-app-travelmate/luasrc/model/cbi/travelmate/wifi_add.lua

```diff
@@ -140,13 +140,18 @@ end
 local login_section = (m.hidden.ssid or "") .. (m.hidden.bssid or "")
 login_section = login_section:gsub("[^%w_]", "_")
 local cmd = uci:get("travelmate", login_section, "command")
+local cmd_args_default = uci:get("travelmate", login_section, "command_args")
 cmd_list = m:field(ListValue, "cmdlist", translate("Auto Login Script"),
 	translate("External script reference which will be called for automated captive portal logins."))
-cmd_list:value("none")
+cmd_args = m:field(Value, "cmdargs", translate("Optional Arguments"),
+	translate("Optional additional arguments passed to the Auto Login Script, i.e. username and password"))
```

Comment:
Please change the description to "Space separated list of additional arguments passed to the Auto Login Script, i.e. username and password"